packages/litexa-html/README.md

```diff
@@ -44,7 +44,119 @@ project_dir
         └── html
 ```
 
-## Adding 'mark' SSML Tags
+## Alexa Web API Interface
+
+Integrating your skill with the Alexa Web API will allow it to both send and receive data
+from your web app.
+
+### Skill Manifest
+
+Before your skill can send and receive data through the Alexa Web API, you will need to
+specify the interface in your skill's manifest. You will need to include
+`ALEXA_PRESENTATION_HTML` in your `skill.*` file to let Alexa know that you are using it.
```

Comment:
I believe the extension does this for you - in fact, if you globally install this one, it will add the HTML interface to every skill, even if you're not using it. Might be good to recommend making this extension a project-specific installation.

Reply:
I think it's good to just have users explicitly check their `skill.*` for the interface, so that they get used to looking at that manifest file + knowing where interfaces are added to it.